packages/authentication/package.json

```diff
@@ -22,6 +22,7 @@
     "@loopback/core": "^4.0.0-alpha.6",
     "@types/passport": "^0.3.3",
     "passport": "^0.3.2",
+    "passport-http": "^0.3.0",
```

Comment:
Please move this line to `devDependencies`. We should allow LoopBack users/applications to decide which version of `passport-http` they want to use; also AFAICT this new dependency is used by tests only.